app/dashboard/courses/[courseId]/VideoList.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+import { getCourseVideosById } from "@/app/api/courses/courses.api";
+import { convertSecondsToMinutes } from "@/app/configs/utils/format";
+import { useQuery } from "@tanstack/react-query";
+
+export const VideoList = ({ id }: { id: string }) => {
+  const { data: videos } = useQuery({
+    queryKey: ["videos", id],
+    queryFn: () => getCourseVideosById(id),
+    enabled: Boolean(id),
+  });
+
+  const sortedVideosByOrder = videos?.sort((a, b) => a.order - b.order);
```

Comment:
as I remember firebase can sort data for you, maybe you don't need to do it here